pages/docs/couple-your-code/couple-your-code-waveform.md

```diff
@@ -40,17 +40,13 @@ If the Dirichlet participant $$\mathcal{D}$$ calls `readBlockVectorData`, it sam
 
 ## Experimental API for waveform iteration
 
-If we want to improve the accuracy by using waveforms, this requires an extension of the existing API, because we need a way to tell preCICE where we want to sample the waveform. For this purpose, preCICE offers an experimental API. Here, `readBlockVectorData` accepts an additional argument `relativeReadTime`. This allows us to choose the time where the waveform should be sampled:
+If we want to improve the accuracy by using waveforms, this requires you to tell preCICE where we want to sample the waveform. For this purpose, preCICE offers the argument `relativeReadTime` for all read data functions of the API:
```

Comment:
Passing relative read time is no longer a choice.
This entire paragraph needs rewording to reflect this.

A visual example could be helpful here.

Reply:
I reformulated a bit via [d280ec6](https://github.com/precice/precice.github.io/pull/257/commits/d280ec6866ba1a1720cd6b0bb406c88a8517ba3a). 

One figure I could offer is the following:

![Screenshot from 2023-05-28 15-13-58](https://github.com/precice/precice.github.io/assets/5740604/dc382f28-1252-40f1-a870-6d915337645a)

Does this go into the right direction? I can tweak it a bit. If you have something different in mind: Let's do this in a follow up PR and get this PR merged, because there are some more dependencies (read data functions...).

Reply:
That looks very good. Thr picture looks useful. Could you add it along it's sources?

Reply:
Done. [680eee7](https://github.com/precice/precice.github.io/pull/257/commits/680eee7bf841f44de0d6c9494372099af78d1115)